### PR TITLE
Dala 533 remove index panel

### PR DESCRIPTION
**Pull request by ppuetsch**
# Pull Request 533 Remove Index Panel
Removes the Index-Seleciton Tab Panel on the Company Search UI Page, including removal of associated functionalities.
Additionally: 
DALA-537:
*  Remove Link for image on landing page
* center-align join dataland box
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one E2E Test exists testing the new feature
- [x] ~Documentation is updated as required~
- [x] ~The automated deployment is updated if required~
- [x] The new version is deployed to the dev server using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [x] ~There is at least one picture for each story, which was created before coding has started~
- [x] ~If any work on the UI is to be merged, those changes were also documented in the Figma~
- [x] The local Dev stack still works: execute `startDevelopmentStack.sh`, npm serve, npm teste2e and execute Cypress Tests locally

### CODE REVIEW

File: dataland-frontend/src/components/pages/SearchCompaniesForFrameworkData.vue [hunk 1]
```diff
@@ -158,12 +146,11 @@ export default {
       if (filtered) {
```
Comment (MarcTM01) [inline dataland-frontend/src/components/pages/SearchCompaniesForFrameworkData.vue:146]: On second thought: Da currentSearchbarInput und currentFilteredFrameworks standardmäßig sowiso undefined sind, und nur geändert werden, wenn nötig, könnte man die ganze filtered-Variable (wahrscheinlich) oder einfach weglassen und immer `this.$refs.frameworkDataSearchBar.queryCompany(this.currentSearchBarInput, this.currentFilteredFrameworks)` aufrufen oder?

File: dataland-frontend/src/components/pages/SearchCompaniesForFrameworkData.vue [hunk 2]
```diff
@@ -158,12 +146,11 @@ export default {
...
         this.$refs.frameworkDataSearchBar.queryCompany(this.currentSearchBarInput, this.currentFilteredFrameworks);
       } else {
+        this.$refs.frameworkDataSearchBar.queryCompany("", "");
```
[the remark this answers is not in the document]
Comment (ppuetsch) [reply dataland-frontend/src/components/pages/SearchCompaniesForFrameworkData.vue]: fixed - ich habe in dem zusammenhang die komponente als ts-komponente umgewandelt

File: dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts [hunk 3]
```diff
@@ -200,6 +198,24 @@ describeIf(
       cy.ensureLoggedIn("data_uploader", Cypress.env("KEYCLOAK_UPLOADER_PASSWORD"));
     });
 
+    it(
+      "Upload a company without uploading framework data for it and check if it neither appears in the + " +
+        "autocomplete suggestions nor in the search results, even though no framework filter is set.",
+      () => {
+        const companyName = "ThisCompanyShouldNeverBeFound12349876";
```
Comment (MarcTM01) [inline dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts:205]: Solche sachen wollten wir ja eigentlich aus den FakeFixtures holen oder nicht?

File: dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts [hunk 4]
```diff
@@ -66,47 +59,52 @@ describe("As a user, I expect the search functionality on the /companies page to
...
   });
 
-  it("Company Search by Identifier", () => {
+  it("Execute a company Search by identifier and assure that the company is found", () => {
     cy.visitAndCheckAppMount("/companies");
     const inputValue = companiesWithData[1].companyInformation.identifiers[0].identifierValue;
-    executeCompanySearch(inputValue);
-    verifyTaxonomySearchResultTable();
-    checkViewButtonWorks();
+    const expectedCompanyName = companiesWithData[1].companyInformation.companyName;
```
[the remark this answers is not in the document]
Comment (emanueldfine) [reply dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts:97]: Ist meines Erachtens nicht wichtig. Ändere ich auf 0 um keine Verwirrung zu stiften.

[the remark this answers is not in the document]
Comment (emanueldfine) [reply dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts]: fixed

File: dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts (hunk 3 above)
Comment (emanueldfine) [reply dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts:205]: Hier muss eine Company erstellt werden, deren CompanyName - auch als substring - keiner Company zugehörig ist, für die bereits FrameworkData hochgeladen wurde.
Da durch die Prepopulation jedoch für alle companies auch direkt ein framework dataset hochgeladen wird, kann ich an dieser Stelle keinen companyName verwenden, der ein substring der company names in den fakefixtures ist.

File: dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (MarcTM01) [reply dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts]: Ok

File: dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts (hunk 3 above)
Comment (MarcTM01) [reply dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts:205]: Ok

File: .github/workflows/CI.yaml [hunk 5]
```diff
@@ -138,6 +138,12 @@ jobs:
         with:
           name: lcovResults
           path: ./*.info
+      - name: Upload FE E2E lcov HTML reports
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: lcovHtmlReportsE2E
+          path: ./lcov-reports
```
Comment (emanueldfine) [inline .github/workflows/CI.yaml:146]: ?

File: build.gradle.kts [hunk 6]
```diff
@@ -88,8 +88,8 @@ sonarqube {
             "**/test/**," +
                 "**/tests/**," +
                 "**/LocalCorsConfig.kt," +
-                "./dataland-frontend/src/main.ts" +
-                "./dataland-frontend/src/components/helper/*"
+                "./dataland-frontend/src/main.ts," +
+                "./dataland-frontend/build"
```
Comment (emanueldfine) [inline build.gradle.kts:92]: ?

File: build.gradle.kts [hunk 7]
```diff
@@ -104,6 +104,8 @@ jacoco {
 
 tasks.jacocoTestReport {
     dependsOn(tasks.build)
+    dependsOn(tasks.getByPath(":dataland-backend:compileKotlin"))
+    dependsOn(tasks.getByPath(":dataland-csvconverter:compileKotlin"))
```
Comment (emanueldfine) [inline build.gradle.kts:108]: ?

File: build.gradle.kts (hunk 6 above)
Comment (emanueldfine) [reply build.gradle.kts:92]: Ausprobieren ob build Zeile raus kann.

File: .github/workflows/CI.yaml (hunk 5 above)
Comment (emanueldfine) [reply .github/workflows/CI.yaml:146]: rausnehmen (es handelte sich um html files um coverage reports im browser anzuschauen)

**Pull request merged**